robotidy/transformers.py

```diff
@@ -398,6 +399,8 @@ class NormalizeSectionHeaderName(ModelTransformer):
     Robot Framework is quite flexible with the section header naming. Therefore ``*setting`` or ``*** SETTINGS`` are
     both recognized as ``*** Settings ***``. This transformer normalize naming to follow ``*** SectionName ***`` format
     (with plurar variant). Optional data after section header (for example data driven column names) is preserved.
+    It is possible to upper case section header names by passing uppercase=True parameter.
```

Comment:
I would enclose all code-related words and variables inside code block, for better display.

Reply:
I've updated docs a bit